samples/basic/sampleApp/src/main.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <devicetree.h>
+#include <drivers/gpio.h>
+#include <net/net_if.h>
+#include <net/socket.h>
+#include <net/socket_net_mgmt.h>
+#include <net/net_core.h>
+
+#include <drivers/modem/ublox-sara-n310.h>
+
+/* 1000 msec = 1 sec */
+#define SLEEP_TIME_MS 1000
+
+/* The devicetree node identifier for the "led0" alias. */
+#define LED0_NODE DT_ALIAS(led0)
+
+/* device label necessary for binding */
+#define N310_DEVICE_LABEL DT_LABEL(DT_INST(0, ublox_saran310))
+
+#if DT_NODE_HAS_STATUS(LED0_NODE, okay)
+#define LED0 DT_GPIO_LABEL(LED0_NODE, gpios)
+#define PIN DT_GPIO_PIN(LED0_NODE, gpios)
+#define FLAGS DT_GPIO_FLAGS(LED0_NODE, gpios)
+#else
+/* A build error here means your board isn't set up to blink an LED. */
+#error "Unsupported board: led0 devicetree alias is not defined"
+#define LED0 ""
+#define PIN 0
+#define FLAGS 0
+#endif
+
+static const struct device *dev;
```

Comment:
would make sense to change the variable names to things they will represent, i.e. `static const struct device *dev_ubloxn3`